src/features/main/Start.js

```diff
@@ -0,0 +1,69 @@
+import PropTypes from "prop-types";
+import React from "react";
+import styled from "styled-components";
+
+import startIcon from "../../assets/icon-start.svg";
+
+const StyledStart = styled.div`
+  width: 100%;
+  height: 100%;
+  min-height: 568px;
+  background-color: #0000008e;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  .wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+  }
+  .startBtn {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+  }
+  .description {
+    width: 100%;
+    font-size: small;
+    text-align: center;
+    color: #bc955c;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+`;
+
+function Start({ onClick }) {
+  return (
+    <StyledStart>
+      <div className="wrapper">
+        <div
+          className="startBtn"
```

Comment:
```suggestion
          className="start-btn"
```

개인적으로 저는 CSS 클래스 네이밍은 케밥 케이스로 진행 하고 있습니다. 그리고 왠만하면 btn이라는 축약어 보다는 풀네이밍으로 button을 써주는 편이에요. 
이유는 다음과 같습니다. 

https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/

![image](https://user-images.githubusercontent.com/72188256/153691867-8c8e89ff-0d3b-4c41-9899-7ba2b3f95a94.png)

Reply:
아! 이 부분은 제가 놓쳤네요;; 수정하겠습니다~